include/exadg/incompressible_navier_stokes/user_interface/parameters.cpp

```diff
@@ -634,14 +618,13 @@ Parameters::implicit_convective_problem() const
 bool
 Parameters::nonlinear_viscous_problem() const
 {
-  return viscous_problem() and viscous_term_is_nonlinear();
+  return (viscous_problem() and viscous_term_is_nonlinear());
```

Comment:
We should check whether we really need the function `viscous_term_is_nonlinear()`, or whether we could just insert the two lines of this function here. The use of `viscous_term_is_nonlinear()` does not seem to be intuitive in the temporal/spatial discretization routines.

Reply:
now it is only used in the `parameters.h/.cpp`, I can remove it.